frappe/email/receive.py

```diff
@@ -321,6 +328,46 @@ def process_part(self, part):
 		if part.get_filename():
 			self.get_attachment(part, charset)
 
+	def get_attached_message(self,part):
+		list=[None]*4
+		list[0]=part.__getitem__('from')
```

Comment:
the problem with this approach which i had to disregard is that for test case 2 the content is still looped through and added if you add the class is it will process all the info and add it and then display the content again after